lib/associations/belongs-to-many.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var _ = require('lodash');
+
+module.exports = function(Resource, resource, association) {
+  // access points
+  var subResourceName =
+    association.target.options.name.plural.toLowerCase();
+
+  // Find reverse association
+  var associationPaired;
```

Comment:
No, I think It's explicit :)

See https://github.com/dchester/epilogue/pull/68